src/components/service.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+    font-family: 'Inter';
+    src: local('Inter'), url(../fonts/Inter/Inter-VariableFont_slnt,wght.ttf) format('truetype');
+    font-weight: 100 1000;
+	font-stretch: 25% 151%
+  }
+
+div{
+    font-family: Inter;
+    font-weight: 350;
+    text-align: left;
+}
+
+.heading{
+    margin: auto;
+    width: 50%;
+    padding: 10px;
+    text-align: center;
+    top: 100px;
+    color: rgb(255, 255, 255);
+}
+
+.textBackground {
+    padding: 20px;
+    position: relative;
+    max-width: 1000px;
+    width: 80%;
+    background: rgb(33, 36, 41);
+    box-shadow: rgb(0 0 0 / 1%) 0px 0px 1px, rgb(0 0 0 / 4%) 0px 4px 8px, rgb(0 0 0 / 4%) 0px 16px 24px, rgb(0 0 0 / 1%) 0px 24px 32px;
+    border-radius: 30px;
+    top: 3vh;
+    height: 27vh;
+    margin: auto;
+    color: rgb(195, 197, 203);
+}
+
+.serviceBackground{
+    background-color: rgb(44, 47, 54);
+    min-height: 100vh;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: radial-gradient(50% 50% at 50% 50%, rgba(33, 114, 229, 0.1) 0%, rgba(33, 36, 41, 0) 100%);
+    background-size: 'cover';
+
+}
+
+.tryButton {
+    padding: 18px;
+    max-width: 700px;
+    width: 70%;
+    text-align: center;
+    border-radius: 20px;
+    outline: none;
+    border: 1px solid transparent;
+    text-decoration: none;
+    display: flex;
+    -webkit-box-pack: center;
+    justify-content: center;
+    flex-wrap: nowrap;
+    -webkit-box-align: center;
+    align-items: center;
+    cursor: pointer;
+    position: relative;
+    z-index: 1;
+    background-color: rgba(21, 61, 111, 0.44);
+    color: rgb(109, 168, 255);
+    font-size: 16px;
+    font-weight: 500;
+    margin: auto;
+    bottom: 0px;
+}
+
+.tryButton:hover{
+    background-color: rgba(19, 54, 98, 0.44);
+}
+
+
+
```

Comment:
Too many newlines at the end, one is enough

Reply:
Also covered by general formatting.